docusaurus/docs/how-to-guides/panel-plugins/plugin-internationalization.md

```diff
@@ -0,0 +1,306 @@
+           },
+           {
+             value: 'lg',
+-            label: 'Large',
++            label: t('panel.options.seriesCountSize.options.lg', 'Large'),
+           },
+         ],
+       },
+```
+
+Example using the `Trans` component:
+```diff
+import { SimpleOptions } from 'types';
+ import { css, cx } from '@emotion/css';
+ import { useStyles2, useTheme2 } from '@grafana/ui';
+ import { PanelDataErrorView } from '@grafana/runtime';
++import { Trans } from '@grafana/i18n';
+ 
+ interface Props extends PanelProps<SimpleOptions> {}
+ 
+@@ -60,9 +61,15 @@ export const SimplePanel: React.FC<Props> = ({ options, data, width, height, fie
+ 
+       <div className={styles.textBox}>
+         {options.showSeriesCount && (
+-          <div data-testid="simple-panel-series-counter">Number of series: {data.series.length}</div>
++          <div data-testid="simple-panel-series-counter">
++            <Trans i18nKey="components.simpePanel.options.showSeriesCount">
++              Number of series: {{ numberOfSeries: data.series.length }}
++            </Trans>
++          </div>
+         )}
+-        <div>Text option value: {options.text}</div>
++        <Trans i18nKey="components.simpePanel.options.textOptionValue">
++          Text option value: {{ optionValue: options.text }}
++        </Trans>
+       </div>
+     </div>
+   );
+```
+
+## Obtain the translated text
+
+Use the `i18next` [parser](https://github.com/i18next/i18next-parser#readme) and [exta] to sweep all input files, extract tagged `i18n` keys, and save the translations. 
+
+### Parse for translations
+
+Install the `i18next` parser:
+
+```shell npm2yarn
+npm install i18next-parser
+```
+
+Next, create a configuration file `src/locales/i18next-parser.config.js` and configure it so the parser sweeps your plugin and extracts the translations into the `locales/[$LOCALE]/[your-plugin].json`:
+
+:::danger
+The path `locales/[$LOCALE]/[your-plugin].json` is mandatory. If you modify it translations won't work.
+:::
+
+```js title="i18next-parser.config.js"
+const pluginJson = require('../plugin.json');
+
+module.exports = {
+  locales: ['en-US'], // Only en-US  is updated - Crowdin will PR with other languages
+  sort: true,
+  createOldCatalogs: false,
+  failOnWarnings: true,
+  verbose: false,
+  resetDefaultValueLocale: 'en-US', // Updates extracted values when they change in code
+  defaultNamespace: pluginJson.id,
+  input: ['../**/*.{tsx,ts}'],
+  output: 'src/locales/$LOCALE/$NAMESPACE.json',
+};
+```
+
+### Obtain your translation file
+
+Add the translation script `i18n-extract` to `package.json`:
+
+```json title="package.json"
+  "scripts": {
+    "i18n-extract": "i18next --config src/locales/i18next-parser.config.js",
+  },    
+```
+
+Run the script to translate the files:
+
+```shell npm2yarn
+npm run i18n-extract
+```
+
+The translation file will look similar to this:
+
+```
+{
+  "components": {
+    "simpePanel": {
```

Comment:
```suggestion
    "simplePanel": {
```

Reply:
Great catch Jack 🙌 , that needs to change in the example code as well and the json output too